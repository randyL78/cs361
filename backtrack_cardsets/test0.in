1 2 3
3 4 5
1 2 4
1 4 6
3 5 6
