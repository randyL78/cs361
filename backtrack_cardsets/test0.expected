2 packets required for a full collection.
